armi/physics/neutronics/const.py

```diff
@@ -52,3 +55,28 @@
 RESTARTFILES = "Restart files"
 INPUTOUTPUT = "Input/Output"
 FLUXFILES = "Flux files"
+
+
+class SolutionType(Enum):
+    """Type of flux computed."""
+
+    REAL = 0
+    ADJOINT = 1
+    REAL_AND_ADJOINT = 2
```

Comment:
can you make these 1, 2 and 3? It would let us transition a bitfield and literally treat REAL_AND_ADJOINT as REAL & ADJOINT, as in Flags. Imagine a code that doesnt know how to do both at the same time, it wouldnt need a special case to do both, it could do if REAL in solType, do real solve; if ADJOINT in solType do adjoint solve;

Reply:
or make this a flags, with auto()?

Reply:
I made a few code-specifics ones of these in an internal library and they correspond to specific values. Maybe I could move this back out of the framework because you're right about it not being great for flags.

Reply:
Pick one. If we do it on the framework side, we need to do it right. Plugins can do what they want. Flags/bitfield is 100% the way to go. Maybe we can address this in a separate PR?